apps/web/i18n/routes.tsx

```diff
@@ -0,0 +1,109 @@
+import { Locale, defaultLanguage } from './I18n'
+
+export interface AnchorAttributes
+  extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
+  href: string
+  as: string
+}
+
+export type ContentType =
+  | 'article'
+  | 'page'
+  | 'category'
+  | 'contentcategory'
+  | 'articlecategory'
+  | 'news'
+  | 'search'
+  | 'lifeevent'
+  | 'lifeeventpage'
+  | 'adgerdir'
+  | 'vidspyrna-frontpage'
+
+export const routes: Record<ContentType, Record<Locale, AnchorAttributes>> = {
+  article: {
+    is: { as: '/[slug]', href: '/[slug]' },
+    en: { as: '/en/[slug]', href: '/en/[slug]' },
+  },
+  page: {
+    is: { as: '/stofnanir/[slug]', href: '/stofnanir/stafraent-island' }, // TODO FOLDER STRUCTURE NEEDS FIXING
```

Comment:
This looks like the only instance where `as !== href`, maybe we can simplify the `routes` mapping object by changing `{as: string, href: string}`to string e.g. `{is: '/[slug]', en: '/en/[slug]'}`.
Then we can handle the temporary edgecase of `/stofnanir/stafraent-island` inside the resolver function